socorro/external/rabbitmq/crashstorage.py

```diff
@@ -223,29 +220,28 @@ def _ack_crash(self, connection, crash_id, acknowledgement_token):
 
 class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
     required_config = Namespace()
-    required_config.routing_key = change_default(
-        RabbitMQCrashStorage,
+    required_config.add_option(
         'routing_key',
-        'socorro.reprocessing'
+        default='socorro.reprocessing',
+        doc='the name of the queue to recieve crashes',
+        reference_value_from='resource.rabbitmq',
     )
-    required_config.filter_on_legacy_processing = change_default(
-        RabbitMQCrashStorage,
+    required_config.add_option(
         'filter_on_legacy_processing',
-        False
+        default=False,
+        doc='toggle for using or ignoring the throttling flag',
+        reference_value_from='resource.rabbitmq',
     )
 
 
 class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
     required_config = Namespace()
-    required_config.rabbitmq_class = change_default(
-        RabbitMQCrashStorage,
+    required_config.add_option(
         'rabbitmq_class',
-        ConnectionContext,
-    )
-    required_config.routing_key = change_default(
-        RabbitMQCrashStorage,
-        'routing_key',
-        'socorro.reprocessing'
```

Comment:
We don't need this since the parent class already has a `routing_key` of `socorro.reprocessing`.